### ISSUE TITLE
FDSNWS: Add data availability query

### ISSUE THREAD

**Issue by megies** [krischer/jane#31]
Would be nice to have the ability to query the server via FDSNWS for availability of data, only doing a quick database query which should be very fast. This could be useful for use together with e.g. MSNoise (if we or somebody else is adding FDSNWS support to it; CC @ThomasLecocq).

With FDSNWS I guess this would mean implementing support for the `matchtimeseries` option in the station query..

Basically that should be more or less copy/paste from the dataselect query and just leave away the actually reading the data from file.
